kolibri/logger/signals.py

```diff
@@ -0,0 +1,19 @@
+import logging
+
+from django.contrib.auth.signals import user_logged_in
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from .models import ContentSessionLog, UserSessionLog
+
+logger = logging.getLogger(__name__)
+
+@receiver(user_logged_in)
+def login_session_receiver(sender, **kwargs):
+    user = kwargs["user"]
+    UserSessionLog.update_log(user)
+
+@receiver(post_save, sender=ContentSessionLog)
```

Comment:
Update when a ContentSessionLog is saved, means that they were doing something, but if it was `passive` like watching a video, we might not know about it from the heartbeat code.